ldap_probe/tasks.py

```diff
@@ -258,3 +260,51 @@ def trim_ad_duplicates():
         except Exception as error:
             LOG.error(error)
             raise error
+
+
+@shared_task(queue='email', rate_limit='1/s')
+def raise_ldap_probe_failed_alert(instance_pk=None, subscription=None):
```

Comment:
Why do we supply a default value for instance_pk if it's an error not to supply one?
```suggestion
def raise_ldap_probe_failed_alert(instance_pk, subscription=None):
```

Reply:
Using a default for instance_pk makes for a cleaner error catching mechanism. 
i have no idea what happens if you call ```model.objects.filter()``` without a value in the filter. with this approach, i am actually telling the caller what they did wrong.
further more, we call of the task before any database operations are attempted. database ops are expensive; if we can avoid them, we should

Reply:
The only difference with my suggestion would be on the python side. The method call would raise a `TypeError` because it's missing a required argument, rather than running any of the function code at all.